contracts/mainnet/l1_contract.sol

```diff
@@ -55,31 +107,21 @@ contract L1_CONTRACT {
     function publishFromL1() public {
         uint256[] memory payload = new uint256[](11);
         payload[0] = uint256(uint160(address(this)));
-        payload[1] = 10703902247957299200;
+        payload[1] = 10703902247957200;
```

Comment:
a comment to explain those values?

Reply:
@joshualyguessennd if you can remove these hardcoded values and add add them as params to your function it would be best.